app/src/main/java/org/nieghborhoodbikeworks/nbw/MainActivity.java

```diff
@@ -95,7 +98,12 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                         break;
                     }
                     case R.id.nav_queue: {
-                        mNavController.navigate(R.id.queueFragment);
+                        if(mUser.isAdmin()) {
```

Comment:
what happens if `mUser == null`?

Reply:
Good question--I think what i wanted to do was rethink the navigation drawer. Basically, if the user is signed in, show them the nav drawer (i.e. give them access to it). If they aren’t, don’t.

Reply:
I prefer the 1st option because then they won't have to bother to see all these other options they don't actually have access to.